_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -42,6 +42,8 @@ For each test target, one of the following is true:
 - **semantic Role**: the test target is an [inherited][], [supported][], or [required][] [state][] or [property][] of the [semantic role][] of the element on which the test target is specified; or
 - **language feature**: the test target is specified on an [HTML element][namespaced element] and is allowed on that element. Which ARIA states or properties may be used on which element is described in [ARIA in HTML](https://w3c.github.io/html-aria/).
 
+For each test target there are no prohibited attributes for the target's role.
```

Comment:
1. In this rule, the "test targets" are the ARIA states and properties, not the element. So they do not have a role. This should be phrased as "the test target is not prohibited on the role of the element on which it is specified" or something like that (see the way the second bullet is phrased).
2. This sounds like a completely different expectation, should they be numbered as "Expectation 1", "Expectation 2" instead?
3. What is a "prohibited attribute"? This needs to link to some specification. "role" should also link to our definition of "semantic role".